.github/workflows/test-rc-release.yaml

```diff
@@ -25,8 +27,73 @@ on:  # yamllint disable-line rule:truthy
         default: 'main'
 
 jobs:
+  check-rc-testing-annocement:
+    runs-on: 'ubuntu-20.04'
+    if: github.event_name == 'schedule'
+    steps:
+      - name: Checkout apache-airflow
+        uses: actions/checkout@v3
+        with:
+          repository: 'apache/airflow'
+
+      - name: Parse the latest 100 GitHub issues from apache-airflow to check providers testing annocement
+        id: parse-airflow-gh-issues
+        run: |
+          # The default limit is 30. Set it to 100 for retrieving more issues
+          rc_issue_url=`gh issue list \
+            --json createdAt,title,url \
+            --limit 100 \
+            --jq 'map(
+                    select(
+                      .title |
+                      contains ("Status of testing Providers that were prepared on ")
+                    )
+                  ) | .[0].url'`
+
+          echo "rc_issue_url=$rc_issue_url" >> $GITHUB_OUTPUT
+
+      - name: Checkout current repo
+        uses: actions/checkout@v3
+        if: steps.parse-airflow-gh-issues.outputs.rc_issue_url != ''
+
+      - name: Parse the latest GitHub issues for checking existing RC provider testing issue
+        id: parse-current-repo
+        if: steps.parse-airflow-gh-issues.outputs.rc_issue_url != ''
+        run: |
+          # The default limit is 30. Set it to 100 for retrieving more issues
+          rc_issue_url="${{ steps.parse-airflow-gh-issues.outputs.rc_issue_url }}"
+          testing_issue_url=`gh pr list \
```

Comment:
I think this only lists open PRs. Once will close the PRs, and since the issue could still be there in apache-airflow in top 100 issues, are we at risk of recreating PRs? I think we should list here merged PRs too

Reply:
Yes, that makes sense. I waited for the airflow issue to close before I closes the PR in the previous releases. Let me add it 💪 Thanks!

Reply:
just added